docs/credentials.rst

```diff
@@ -0,0 +1,6 @@
+Credentials
```

Comment:
We should leave Credentials out of docs, since we've just removed it from the API.

Reply:
Done.